samples/azure-quantum/resource-estimation/estimation-factoring.ipynb

```diff
@@ -69,41 +70,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def logical_counts(\n",
-    "    qubit_count: int = 0,\n",
-    "    t_count: int = 0,\n",
-    "    rotation_count: int = 0,\n",
-    "    rotation_depth: int = 0,\n",
-    "    ccz_count: int = 0,\n",
-    "    measurement_count: int = 0\n",
-    "):\n",
-    "    import textwrap\n",
-    "\n",
-    "    ir = f\"\"\"\n",
-    "        %Array = type opaque\n",
-    "        %Qubit = type opaque\n",
-    "        attributes #1 = {{ \"entry_point\" }}\n",
-    "        declare %Array* @__quantum__rt__qubit_allocate_array(i64)\n",
-    "        declare void @__quantum__rt__qubit_release_array(%Array*)\n",
-    "        declare void @__quantum__qis__applyunimplemented__body(i64, i64, i64, i64, i64, %Array*)\n",
-    "        define void @Project__Program() #1 {{\n",
-    "        entry:\n",
-    "            %target = call %Array* @__quantum__rt__qubit_allocate_array(i64 {qubit_count})\n",
-    "            call void @__quantum__qis__applyunimplemented__body(i64 {t_count}, i64 {rotation_count}, i64 {rotation_depth}, i64 {ccz_count}, i64 {measurement_count}, %Array* %target)\n",
-    "            call void @__quantum__rt__qubit_release_array(%Array* %target)\n",
-    "            ret void\n",
-    "        }}\n",
-    "        \"\"\"\n",
-    "\n",
-    "    # Support code to transition from pyqir-generator to pyqir package\n",
-    "    try:\n",
-    "        from pyqir.generator import ir_to_bitcode\n",
-    "    except:\n",
-    "        def ir_to_bitcode(ir):\n",
-    "            from pyqir import Context, Module\n",
-    "            return Module.from_ir(Context(), ir).bitcode\n",
-    "\n",
-    "    return ir_to_bitcode(textwrap.dedent(ir))"
+    "%%qsharp\n",
+    "open Microsoft.Quantum.ResourceEstimation;\n",
+    "\n",
+    "operation FactoringFromLogicalCounts() : Unit {\n",
+    "    use qubits = Qubit[12581];\n",
+    "    \n",
+    "    AccountForEstimates([TCount(12), RotationCount(12), RotationDepth(12), CczCount(3731607428), MeasurementCount(1078154040)], PSSPCLayout(), qubits);\n",
```

Comment:
When I look at this line in the notebook it doesn't fit in the width and doesn't wrap and doesn't allow me to scroll. Suggest wrapping it the source code.

Reply:
Thanks, I wrapped it in source code.